src/chart/init.js

```diff
@@ -17,6 +17,7 @@ export default function init(data, test = false) {
     this.setDefaults();
 
     this.raw_data = data;
+    this.initial_data = data;
```

Comment:
Cool.

Reply:
Although... raw_data is never overwritten.  I'm not sure initial_data does anything for us here.

Reply:
When a filter changes the chart reverts back to `this.raw_data`.  The data passed to `chart.draw()` isn't attached to the chart object but it defines everything downstream, e.g. `chart.filtered_data`, mark data, etc.

Reply:
This is to help in situations where customization is needed and a callback (usually, `preprocess`) modifies raw_data, now the modified raw_data can be compared to initial_data instead of having to define a custom `super_raw_data` or something similar in the `init` callback.